### PR TITLE
Editorial: markdown cleanup + small fixes

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #60 #58

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,319 +4,306 @@
...
+  const reader = port.in.getReader();
 
-<section data-dfn-for="Navigator" data-link-for="Navigator">
-<h2><dfn>Navigator</dfn> partial interface</h2>
-<pre class="idl">
-[Exposed=Window, SecureContext]
-partial interface Navigator {
-  [SameObject] readonly attribute Serial serial;
-};
+  for await (const { done, data } of reader.read()) {
```
Comment (marcoscaceres) [inline index.html]: @reillyeon I didn't actually try this... but I assume it works?

**Comment by reillyeon** [comment]
I've made some of this cleanups in the process of expanding on the API in my (fork)[https://github.com/reillyeon/serial]. This might be good time to merge that.

**Comment by marcoscaceres** [comment]
@reillyeon, I'm not sure how you want to proceed with this PR?

**Comment by marcoscaceres** [comment]
⛄️🎄 Gentle xmas-themed ping  🎁🎅🏽

**Pull request merged**